### ISSUE TITLE
Open in could not communicate with the outside world

### ISSUE THREAD

**Issue by RasmusKallas** [darktrojan/openwith#234]
I ran the python script with both python 2 and 3 (checking in between) on Pop!_OS 18.04 without any output, but it doesn't work.

**Comment by gerroon** [comment]
Yeah suddenly I am getting this. This ws working fine. Why is it trying to connect to the open outside world when I do not even use it?

**Comment by vc7ee** [comment]
Possible cause: browsers being run in multi-layer sandboxes.
